src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java

```diff
@@ -702,7 +702,9 @@ private GHTeam getOrCreateRepoLocalTeam(GHOrganization org, GHRepository r) thro
         if (t==null) {
             t = org.createTeam(teamName, Permission.ADMIN, r);
         } else {
-            t.add(r);
+            if (!t.getRepositories().containsValue(r)) {
```

Comment:
+1 with possibly the link to the GH doc about that to help possibly (again) in the future.

I suppose this is around https://help.github.com/articles/migrating-your-previous-admin-teams-to-the-improved-organization-permissions/ though a bit fluffy I may understand why @daniel-beck wouldn't want to do so.